projects/core/src/lib/mentee-page/mentee-page.component.scss

```diff
@@ -0,0 +1,22 @@
+@import url("https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400&&display=swap");
+:host {
```

Comment:
instead of host you can add main block .mentee-page and apply these styles

Reply:
done